### PR TITLE
Fix different types in rsub caused bug

### PR DESCRIPTION
**Pull request by zrphercule**
Before this pr, rsub did not convert two elements into the same dtype, therefore "1 - x" may export to an onnx model that two elements of rsub having different dtype.
By adding this symbolic patch this bug should be fixed.
Related test cases also created.

### CODE REVIEW

**Review by houseroad — COMMENTED** [review]
Do we really need Linear and Sigmoid in this case?

**Comment by zrphercule** [comment]
@houseroad  actually no, just copying my local testcase to it... It is fine if we only use a simple "1 - x'

**Comment by houseroad** [comment]
Yeah, let's do it to make it easier to understand.

File: test/onnx/test_pytorch_onnx_caffe2.py [hunk 1]
```diff
@@ -1001,6 +1001,14 @@ def test_group_norm(self):
         model = nn.GroupNorm(3, 6)
         self.run_model_test(model, train=True, input=c, batch_size=BATCH_SIZE)
 
+    def test_rsub(self):
+        class RsubMod(torch.nn.Module):
```
Comment (houseroad) [inline test/onnx/test_pytorch_onnx_caffe2.py]: RsubModel?

**Review by houseroad — APPROVED** [review]
Looks good, thanks!

File: test/onnx/test_pytorch_onnx_caffe2.py (hunk 1 above)
Comment (zrphercule) [reply test/onnx/test_pytorch_onnx_caffe2.py]: Just noticed other "Mod" does not mean "Model" but means "ExportMod"... I will change its name.
